src/Illuminate/Console/Scheduling/ScheduleWorkCommand.php

```diff
@@ -47,7 +47,7 @@ public function handle()
         [$lastExecutionStartedAt, $keyOfLastExecutionWithOutput, $executions] = [null, null, []];
 
         while (true) {
-            usleep(100 * 1000);
+            usleep(500 * 1000); // 0.5 seconds
```

Comment:
I guess this is a good opportunity to move to "Numeric Literal Separator" instead of using a multiplication for thousands notation.

~~~php
usleep(500_000);
~~~

reference: https://wiki.php.net/rfc/numeric_literal_separator

By the way, I guess it is less than a second so the next if clause can be more precise as it checks for the "zero-th" second.